ecs-agent/netlib/network_builder.go

```diff
@@ -216,25 +219,51 @@ func (nb *networkBuilder) configureNetNSInterfaces(ctx context.Context, netNS *t
 		logger.Debug("Configuring interface", logFields)
 		iface.DesiredStatus = netNS.DesiredState
 
-		err := nb.platformAPI.ConfigureInterface(ctx, netNS.Path, iface)
+		err := nb.platformAPI.ConfigureInterface(ctx, netNS.Path, iface, nb.networkDAO)
 		if err != nil {
-			return err
+			if netNS.DesiredState == status.NetworkDeleted {
+				logger.Error(fmt.Sprintf("Failed to delete interface: %v", err), logFields)
```

Comment:
I think you are trying to say - Failed to configure interface instead of `delete`?

Reply:
That is a fair suggestion. I can add a field in the log message to state the desired status and say configure here.